diff --git a/node_modules/@walletconnect/react-native-dapp/dist/providers/WalletConnectProvider.js b/node_modules/@walletconnect/react-native-dapp/dist/providers/WalletConnectProvider.js
index 9b5b1e2..e048399 100644
--- a/node_modules/@walletconnect/react-native-dapp/dist/providers/WalletConnectProvider.js
+++ b/node_modules/@walletconnect/react-native-dapp/dist/providers/WalletConnectProvider.js
@@ -23,7 +23,8 @@ export default function WalletConnectProvider({ children, renderQrcodeModal: may
         if (Platform.OS === 'android') {
             const canOpenURL = await Linking.canOpenURL(uri);
             if (!canOpenURL) {
-                Linking.openURL('https://walletconnect.org/wallets');
+                //Linking.openURL('https://walletconnect.org/wallets');
+                Linking.openURL('https://explorer.walletconnect.com/');
                 throw new Error('No wallets found.');
             }
             await Linking.openURL(uri);
